include/algorithms/kmeans/kmeans_types.h

```diff
@@ -389,8 +363,53 @@ class DAAL_EXPORT DistributedStep2MasterInput : public InputIface
      */
     services::Status check(const daal::algorithms::Parameter * par, int method) const DAAL_C11_OVERRIDE;
 };
+
 } // namespace interface1
-using interface1::Parameter;
+
+/**
+ * \brief Contains version 2.0 of the Intel(R) Data Analytics Acceleration Library (Intel(R) DAAL) interface.
+ */
+namespace interface2
+{
+/**
+ * <a name="DAAL-STRUCT-ALGORITHMS__KMEANS__PARAMETER"></a>
+ * \brief Parameters for K-Means algorithm
+ * \par Enumerations
+ *      - \ref DistanceType Methods for distance computation
+ *
+ * \snippet kmeans/kmeans_types.h Parameter source code
+ */
+/* [Parameter source code] */
+struct DAAL_EXPORT Parameter : public daal::algorithms::Parameter
+{
+    /**
+     *  Constructs parameters of K-Means algorithm
+     *  \param[in] _nClusters   Number of clusters
+     *  \param[in] _maxIterations Number of iterations
+     */
+    Parameter(size_t _nClusters, size_t _maxIterations);
+
+    /**
+     *  Constructs parameters of K-Means algorithm by copying another parameters of K-Means algorithm
+     *  \param[in] other    Parameters of K-Means algorithm
+     */
+    Parameter(const Parameter & other);
+
+    size_t nClusters;                /*!< Number of clusters */
+    size_t maxIterations;            /*!< Number of iterations */
+    double accuracyThreshold;        /*!< Threshold for the termination of the algorithm */
+    double gamma;                    /*!< Weight used in distance computation for categorical features */
+    DistanceType distanceType;       /*!< Distance used in the algorithm */
+    DAAL_UINT64 resultsToEvaluate;   /*!< 64 bit integer flag that indicates the results to compute */
+    DAAL_DEPRECATED bool assignFlag; /*!< Do data points assignment \DAAL_DEPRECATED */
```

Comment:
Why it's needed for new interface?

Reply:
We are required to support this parameter 'assignFlag' until we delete it. Otherwise, the static BC will be failed for users.